lessons/codap-bar-charts/langs/en-us/index.adoc

```diff
@@ -69,7 +75,7 @@ Once we have a graph of randomly distributed data points, we can organize the da
 Practice manipulating the data by completing @printable-exercise{pages/codap-displays-intro.adoc}.
 
 
-Remind students that categorical data is used to classify, rather than to measure. Only when data is being treated categorically will students be invited to fuse data points to create a bar chart. Quantitative (or numeric) data must measure or compare; it is subject to the laws of arithmetic.
+Remind students that categorical data is used to classify, rather than to measure. Only when data is being treated categorically will students be invited to fuse data points - transforming a dot plot into a bar chart. Quantitative (or numeric) data must measure or compare; it is subject to the laws of arithmetic.
 
 Optional: to dig deeper into bar charts, have students turn to @opt-printable-exercise{bar-chart-notice.adoc}.
```

Comment:
Noting that I updated this file to take advantage of @schanzer 's new fill vertical space code. Might not have caught all pages where this is useful.

> [.FillVerticalSpace, cols="^1a,^1a",options="header"]
> |===
> | What do you Notice? 	| What do you Wonder?
> |						|
> |===

Reply:
Thank you! Good call.